src/pages/iou/request/step/IOURequestStepConfirmation.tsx

```diff
@@ -220,13 +204,11 @@ function IOURequestStepConfirmation({
         [report, transaction, transactionTaxCode, transactionTaxAmount, currentUserPersonalDetails.login, currentUserPersonalDetails.accountID, policy, policyTags, policyCategories],
     );
 
-    /**
-     * @param {Array} selectedParticipants
-     * @param {String} trimmedComment
-     * @param {File} [receiptObj]
-     */
     const trackExpense = useCallback(
-        (selectedParticipants, trimmedComment, receiptObj, gpsPoints) => {
+        (selectedParticipants: Participant[], trimmedComment: string, receiptObj?: Receipt, gpsPoints?: IOU.GpsPoint) => {
+            if (!receiptObj || !report || !transaction) {
```

Comment:
Are you sure that we shoudn't track expense if there is no `receiptObj`?

Reply:
there are some cases where we're giving it undefined receipt, but the problem is that during the next step we're passing this `receiptObj` to `TrackExpenseParams`, which are used for backend communication. I'm not sure if we are allowed to change that interface. You've been working on those files, do you know if it's ok to make it accept undefined receipt?

Reply:
@BrtqKr If you have some doubts let's confirm with the author [of the PR](https://github.com/Expensify/App/pull/37282) where track expense functionality was implemented. Have you tested track expense functionality after your updates? But overall it looks like it shouldn't be required to have receipt for expense tracking